staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -623,6 +637,10 @@ type PersistentVolumeClaimSpec struct {
 	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1
 	// +optional
 	StorageClassName *string `json:"storageClassName,omitempty" protobuf:"bytes,5,opt,name=storageClassName"`
+	// VolumeType defines what type of volume is required by the claim.
```

Comment:
These comments are used to generate user-facing documentation.

https://kubernetes.io/docs/api-reference/v1.7/#persistentvolume-v1-core

They really should reference the json field names, not the Go field names.